api/advanced-topics/remote-extensions.md

```diff
@@ -176,27 +176,27 @@ export function activate(context: vscode.ExtensionContext) {
         vscode.commands.registerCommand('myAmazingExtension.persistWorkspaceData', () => {
 
         // Create the extension's workspace storage folder if it doesn't already exist
-        if (!fs.existsSync(context.storagePath)) {
-            fs.mkdirSync(context.storagePath);
+        if (!fs.existsSync(context.storageUri)) {
+            fs.mkdirSync(context.storageUri);
         }
 
         // Write a file to the workspace storage folder
         fs.writeFileSync(
-            path.join(context.storagePath, 'workspace-data.json'),
+            path.join(context.storageUri, 'workspace-data.json'),
             JSON.stringify({ now: Date.now() }));
     }));
 
     context.subscriptions.push(
         vscode.commands.registerCommand('myAmazingExtension.persistGlobalData', () => {
 
         // Create the extension's global (cross-workspace) folder if it doesn't already exist
-        if (!fs.existsSync(context.globalStoragePath)) {
-            fs.mkdirSync(context.globalStoragePath);
+        if (!fs.existsSync(context.globalStorageUri)) {
+            fs.mkdirSync(context.globalStorageUri);
         }
 
         // Write a file to the global storage folder for the extension
         fs.writeFileSync(
-            path.join(context.globalStoragePath, 'global-data.json'),
+            path.join(context.globalStorageUri, 'global-data.json'),
```

Comment:
These should use `vscode.workspace.fs` instead of the native `fs`. Native `fs` doesn't accept VS Code Uri's, and for web extensions (e.g. vscode.dev) it's not accessible, where `vscode.workspace.fs` is

Reply:
Agree, think the whole sample is a bit outdated. Will see what I can do about it.